### DISCUSSION TITLE
Interactive cards with flow

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3389] [Help: Figma] [answered]
hey! We have quite advanced component in Segment that allows users to switch between different places and content. In the designated red box area, I've emphasized interactive cards that serve as clickable elements. These cards essentially function as tabs, dynamically altering the content below to display specific details for each corresponding step. I'm curious if you have any insights on how we could replicate this functionality using Paste components. Your thoughts and suggestions would be greatly appreciated, thanks 🙏🏻

<img width="1440" alt="Connections - Destinations - Delivery overview - 06" src="https://github.com/twilio-labs/paste/assets/140410476/d0e250b8-1740-48da-b243-9c8c853c558c">

**Comment by nkrantz** [comment]
Hi @marektonik - we don't have a component that would fit this use case out of the box. In order to replicate this exact behavior, you have a few options. You can use [Box](https://paste.twilio.design/primitives/box) and [Text](https://paste.twilio.design/primitives/text) primitives to build and style these custom components. You could also use [Tabs](https://paste.twilio.design/components/tabs) to control which view is shown below the interactive cards. The Tabs component has a [state hook](https://paste.twilio.design/components/tabs#using-state-hooks) that allows you to maintain control which tab is selected, so you could build the cards out of [Reset Buttons](https://paste.twilio.design/components/button#reset-button-styles), or [Cards](https://paste.twilio.design/components/card) with "Details" Buttons inside them, to control the view.

**Comment by SiTaggart** [comment, marked as answer]
Hi there,

After further investigation on this particular design, we would strongly recommend using the [Visual Picker](https://paste.twilio.design/components/visual-picker) which was released roughly around the same time as this discussion was created.

It's not perfect, but after being able to understand the user interaction of this screen further, we believe it is the best affordance we can give a user from an accessibility perspective. The visual Picker allows for a choice select to be made (in a visually appealing manner), whilst also being able to respond to that selection to display different content underneath and provide futher interactive content on each card such as tooltips and links within it. 

Thanks
